Formula/pstoedit.rb

```diff
@@ -18,6 +18,14 @@ class Pstoedit < Formula
   depends_on "imagemagick"
   depends_on "plotutils"
 
+  on_linux do
+    depends_on "gcc" => :build
+  end
+
+  # "You need a C++ compiler, e.g., g++ (newer than 6.0) to compile pstoedit."
+  fails_with gcc: "5"
+  fails_with gcc: "6"
```

Comment:
I thought
```ruby
fails_with gcc: "5"
```
by itself is enough to use `gcc` now?

Reply:
In general yes. What happens is that we build with gcc-5 by default. If it fails we directly use brewed gcc 10. We did not test gcc 6 to 9, so we do not add the fails_with.

In this case we also made an attempt with gcc 6 at one point; and know that it fails with gcc 6 too. So I think in that case it's not wrong to add it, to explicitly disable that gcc version.

Reply:
Is it because a `fails_with gcc: "5"` might still use `gcc-6` if that's the system compiler?

Reply:
Yes, `fails_with gcc: "5"` is not enough. If you have any gcc-6 installed on your system (system compiler, or brewed gcc-6), it will use this instead. Of course it impacts only people installing this particular formula from source on Linux, so probably 1 or 2 people on earth in the next 5 years.